hack/deps

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+if [[ $# -eq 0 || ! -f "${OS_OUTPUT}/deps" ]]; then
+  echo "Generating dependency graph ..." 1>&2
+  mkdir -p "${OS_OUTPUT}"
+  go list -f '{{.ImportPath}}{{.Imports}}' ./pkg/... ./cmd/... | tr '[]' '  ' | 
+    grep -vE '^github.com/openshift/origin/cmd/(service-catalog|cluster-capacity)' |
```

Comment:
hardcoded `openshift/origin` references (and other commands/repos) need to be removed.